his/core/application-integration-importhostdefs.md

```diff
@@ -0,0 +1,366 @@
+
+1. The Input Area dialog will appear. You will will need to check the Input Area of the Program. Select **Next**.
+
+   :::image type="content" source="media/la-newproject-import-hostdef11.png" alt-text="IMS Host transaction input area":::
+
+1. The Output Area dialog will appear. You will will need to check the Output Area of the Program. Select **Next**.
+
+   :::image type="content" source="media/la-newproject-import-hostdef12.png" alt-text="IMS Host transaction output area":::
+
+1. The Return Value dialog will appear. You will will need to check the Return Value field for the Program. Select **Next**.
+
+   :::image type="content" source="media/la-newproject-import-hostdef13.png" alt-text="Return value dialog for host transaction":::
+
+1. The Data Tables and Structures dialog will appear. You will will need to check the groups that will become Data Tables and Structures for the Program. This sample program does not require this setting. Select **Next**.
+
+   :::image type="content" source="media/la-newproject-import-hostdef14.png" alt-text="Data Tables and Structures dialog":::
+
+1. The LL Fields Area dialog will appear. You will will need to check the LL fields that need to be excluded from the transaction. Select **Next**.
+
+   :::image type="content" source="media/la-newproject-import-hostdef15.png" alt-text="LL Fields Area dialog":::
+
+1. The ZZ Fields Area dialog will appear. You will will need to check the ZZ fields that need to be excluded from the transaction. Select **Next**.
+
+   :::image type="content" source="media/la-newproject-import-hostdef16.png" alt-text="ZZ Fields Area dialog":::
+
+1. The TRANCODE Fields Area dialog will appear. You will will need to check the TRANCODE fields that need to be excluded from  the Program. Select **Finish**.
+
+   :::image type="content" source="media/la-newproject-import-hostdef17.png" alt-text="TRANCODE Fields Area dialog":::
+
+   - The Designer will present the metadata artifact generated from the COBOL copybook.
+   
+   :::image type="content" source="media/la-newproject-import-hostdef18.png" alt-text="IMS generated metadata view":::
+   
+   The Designer will also generate a Host Definition for the Copybook. This Host Definition will not include the entire provided copybook but only the fields and datatypes needed for the artifact to interact with the Mainframe Program. In other words, while the sample provided above is an entire program, the HIS Designer will extract only the information needed depending on the selected [Programming Model](choosing-the-appropriate-programming-model1.md).
+   
+   :::image type="content" source="media/la-newproject-import-hostdef19.png" alt-text="Host Definition generated by HIS Designer for Logic Apps":::
+
+1. Select **Save All** to generate the HIDX.
+
+## Importing a COBOL Host File Definition
```

Comment:
I assume this section is for host files? The other sections have (CICS) and (IMS) in their names.

```suggestion
## Importing a COBOL Host File Definition (Host Files)
```

Reply:
@haroldcampos: Please confirm that this edit is correct? Thanks!